python/ray/serve/api.py

```diff
@@ -1235,3 +1169,179 @@ def list_deployments() -> Dict[str, Deployment]:
         )
 
     return deployments
+
+
+def deploy_group(
```

Comment:
I left this out because `deploy_group` isn't an API call. I left it out of the `__init__.py` file, so the only way to access it is through a direct import. However, please let me know if we should still make it public API or at least add the decorator.